homeassistant/components/min_max/config_flow.py

```diff
@@ -72,3 +76,49 @@ class ConfigFlowHandler(SchemaConfigFlowHandler, domain=DOMAIN):
     def async_config_entry_title(self, options: Mapping[str, Any]) -> str:
         """Return config entry title."""
         return cast(str, options["name"]) if "name" in options else ""
+
+    @callback
+    @staticmethod
+    def async_setup_preview(hass: HomeAssistant) -> None:
+        """Set up preview WS API."""
+        data_preview = f"{DOMAIN}_preview"
```

Comment:
Why not do this at the data flow handler level? There is no reason for everyone to store a boolean. It should be generic.

Reply:
This comment is no longer relevant